primedev/scripts/client/scriptbrowserhooks.cpp

```diff
@@ -9,7 +9,8 @@ void, __fastcall, (char* pUrl, char flags))
 // clang-format on
 {
 	bool bIsOriginOverlayEnabledOriginal = *bIsOriginOverlayEnabled;
-	if (flags & 2 && !strncmp(pUrl, "http", 4)) // custom force external browser flag
+	bool isHttp = !strncmp(pUrl, "http://", 7) || !strncmp(pUrl, "https://", 8);
```

Comment:
Does this mean https links currently do not open in external browser?
Because I'm pretty sure it was the case with the parkour mode :-?